tensorflow/compiler/xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -205,6 +216,9 @@ class ParameterizedGemmRewriteTest
     GemmRewriteTest::MatchOptimizedHlo(
         hlo, absl::StrReplaceAll(pattern, replacements_), print_operand_shape);
   }
+  absl::string_view CustomCallTarget() {
+    return replacements_[kCustomCallTargetPlaceholder];
```

Comment:
Comment by @cheshire:

What is the connection of this line to the TF32 change?

Reply:
This is an more or less unrelated fix to the same test. Previously it was assumed that cublasLT was never used in a particular case, but it is now supported. Let me know if you need this to be split off into its own PR.

Reply:
Yes please.

Reply:
Done. See PR https://github.com/tensorflow/tensorflow/pull/59220.